.github/workflows/linters.yml

```diff
@@ -0,0 +1,37 @@
+name: Linters
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  lint-markdown:
+    runs-on: ubuntu-latest
```

Comment:
As this is used for all linters, can the preferred system be set “globally”?

Reply:
I looked I believe is possible but the idea is that each linter is separated and on his own, I will keep it mentioned individually in each linter for now